### PR TITLE
Simplify config setup

### PR DESCRIPTION
**Pull request by ManuelBilbao**
The idea is to centralize all needed config values in one file. There'll be a `config.json` file where all values should be set up. Then, the `setup.sh` script will generate necessary config files and update the `Makefile`.
The steps should be something like:
```
# Create and populate config.json...
bash setup.sh
make all
```

### CODE REVIEW

File: config.example.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+		"web_url": "http://localhost:3002",
+		"wallet_connect_apikey": "",
+		"ankr_token": "",
+		"extra_tokens": [{
+			"l1Address": "0x",
+			"symbol": "LAM",
+			"name": "Lambcoin",
+			"decimals": 18
+		}]
+	}
```
[the remark this answers is not in the document]
Comment (ManuelBilbao) [reply config.example.json:41]: `web_url` is important, other ones are all optional, you can opt not to use them

File: setup.sh [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+# ZK env
+sed "s;l2-inits/custom.init.env;l2-inits/$ZK_ENV.init.env;" configs/network.toml | \
+sed "s/^network.=.*$/network = $(get_param .network.l1.name | lower)/" | \
+sed "s/^fee_account_addr.=.*$/fee_account_addr = $(get_param .network.fee_account_addr)/" | \
+sed "s;^web3_url.=.*$;web3_url = $(get_param .network.l1.rpc_url);" | \
+sed "s/^operator_private_key.=.*$/operator_private_key = $(get_param .network.operator_private_key)/" | \
+sed "s/^operator_commit_eth_addr.=.*$/operator_commit_eth_addr = $(get_param .network.operator_commit_eth_addr)/" | \
+sed "s/^operator_blobs_private_key.=.*$/operator_blobs_private_key = $(get_param .network.operator_blobs_private_key)/" | \
+sed "s/^operator_blobs_eth_addr.=.*$/operator_blobs_eth_addr = $(get_param .network.operator_blobs_eth_addr)/" | \
+sed "s/^fee_account_private_key.=.*$/fee_account_private_key = $(get_param .network.fee_account_private_key)/" > custom_configs/$ZK_ENV.toml
```
[the remark this answers is not in the document]
Comment (ManuelBilbao) [reply setup.sh:65]: As this writes on another file, sponge isn't necessary

**Review by fborello-lambda — APPROVED** [review]
LGTM

**Comment by ManuelBilbao** [comment]
I forgot to update the README with more description about the configs. Let me update it please and hopefully it's clearer what configs are required and what they do

**Comment by fborello-lambda** [comment]
Nice. The README is clear and shows the needed configs.

**Review by ilitteri — APPROVED** [review]
LGTM

**Pull request merged**